current_spring2026/app.py

```
@@ -0,0 +1,492 @@
+            </div>
+            """, unsafe_allow_html=True)
+
+    else:
+        st.markdown(f"""
+        <div class="no-results">
+            <div class="no-results-icon">🗂️</div>
+            <div class="no-results-title">No matching materials found</div>
+            <p>Try rephrasing your query, or use one of the example searches above.<br/>
+            The full collection spans photographs, maps, newspapers, manuscripts, and more.</p>
+        </div>
+        """, unsafe_allow_html=True)
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)